test/integration/root_test.go

```diff
@@ -44,18 +46,22 @@ func TestMain(m *testing.M) {
 }
 
 // Create or return a scan to be shared between tests
-func getRootScan(t *testing.T) (string, string) {
+func getRootScan(t *testing.T, scanTypes ...string) (string, string) {
 	testInstance = t
 
-	if len(rootScanId) > 0 {
+	if len(rootScanId) > 0 && len(scanTypes) == 0 {
 		log.Println("Using the scanID: ", rootScanId)
 		log.Println("Using the projectID: ", rootScanProjectId)
 		return rootScanId, rootScanProjectId
 	}
 
-	rootScanId, rootScanProjectId = createScan(testInstance, Zip, Tags)
-
-	return rootScanId, rootScanProjectId
+	if len(scanTypes) > 0 {
```

Comment:
should we check here for the rootscan len too?
maybe to have one check for the root scanId first than only to check the scan types? (if rootscanid <= 0, return)